pincer/middleware/interaction_create.py

```diff
@@ -20,6 +22,8 @@ def convert_message(self, message: Union[Embed, Message, str]) -> Message:
     """Converts a message to a Message object"""
     if isinstance(message, Embed):
         message = Message(embeds=[message])
+    elif isinstance(message, File) or isinstance(message, Image):
```

Comment:
You can merge the instance call here
```diff
-     elif isinstance(message, File) or isinstance(message, Image):
+    elif isinstance(message, (File, Image)):
```

Reply:
lol, just saw this a second ago in the aiohttp source. Was just about to change it.